arches_lingo/database_routers.py

```diff
@@ -0,0 +1,45 @@
+import random
+import logging
+
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class ArchesRouter:
+    write_database_alias = None
+
+    for alias, database in settings.DATABASES.items():
+        if database.get("WRITE_MODE") is True:
+            write_database_alias = alias
+            break
+
+    read_database_aliases = [
+        alias
+        for alias, _database in settings.DATABASES.items()
+        if _database.get("READ_MODE") is True
+    ]
+
+    def db_for_read(self, model, **hints):
```

Comment:
We should probably prefer the database hinted at if it's in `hints`.  I think this is what's making these databases sticky.  And we want them to be sticky to work around replication lag.

Reply:
See https://github.com/django/django/blob/main/django/db/utils.py